### DISCUSSION TITLE
When you're learning everything for the first time...

### DISCUSSION THREAD

**Discussion by Humjujubee** [community/community#113493] [Codespaces]
So I have many questions but I think what is helping me to learn and understand how this forum works and makes things happen that I never imagined me being a part of something awesome?

**Comment by KidiIT** [comment]
Hello @Humjujubee, 
Firstly, I want to say welcome to GitHub community. GitHub is a great platform for developers and even newbie to host and share their projects or software. The big question here is how ready are you? Well if you’re starting all fresh, I’d advise you pick a good open source or sponsored GitHub developer that suite your preferences it’s just like a mentor who will tutor you to be a good programmer. Though it’s not for free but it’s a good way to start from somewhere. That said, welcome once again and happy coding🫶❤️ from KIDI’S TECH your friend😊

**Comment by AuroraEchos** [comment]
Hey there! 😊

It's fantastic to hear that you're finding this forum to be such a positive and enriching experience! 🎉 Discovering communities that help us learn and grow in unexpected ways is always exciting.

To address your question about how this forum works and how it facilitates such awesome experiences, I'd say it's all about the collective knowledge and passion of its members. This forum serves as a platform for individuals with diverse backgrounds and expertise to come together, share ideas, ask questions, and collaborate on projects.

By actively participating in discussions, asking questions, and contributing your insights, you become an integral part of this vibrant community. It's through this exchange of knowledge and experiences that we can collectively achieve things we never thought possible.

So keep engaging, keep asking, and keep being a part of something awesome! Together, we can continue to learn, grow, and make meaningful contributions to this forum and beyond. 🚀

**Comment by z3dxian6** [comment]
Hello everyone ! 😊 My name is Zoran, I'm 22 years old and I'm a student in network and systems engineering with a specialization in cybersecurity at the ISTY of Paris Saclay University. I'm also developing my skills in ethical hacking through various means such as websites and books :bowtie:. Don't hesitate to contact me and check out my profile!🎉🎉 .
